<launch>
  <!--simulate rgbd camera-->
  <node pkg="dor_control" type="execute_tasks.py" name="task_executor" output="screen"/>
</launch>
